### PR TITLE
Update _index.md

### PR DESCRIPTION
**Pull request by inindev**
Document 'go fmt' step

Signed-off-by: John Clark inindev@gmail.com

### CODE REVIEW

File: content/docs/guides/contributing/_index.md [hunk 1]
```diff
@@ -44,6 +44,7 @@ Here is how to contribute back some code or documentation:
 - Fork repo
 - Create a feature branch off of the `dev` branch
 - Make some useful change
+- Format the change ```go fmt <myfile.go>```
```
Comment (aykevl) [inline content/docs/guides/contributing/_index.md]: We have a helper for this in the Makefile:

```suggestion
- Format the change using `make fmt`
```

**Comment by inindev** [comment]
good to know, thanks
should be all set now

**Comment by ysoldak** [comment]
@deadprogram this looks good to merge

**Comment by deadprogram** [comment]
@inindev could you please rebase this branch off of the `dev` branch? Thanks!

**Pull request closed without merging**